core/domain/stats_domain.py

```diff
@@ -546,17 +545,52 @@ def from_dict(cls, playthrough_dict):
         Returns:
             Playthrough. The corresponding Playthrough domain object.
         """
-        playthrough_actions = [
-            LearnerAction.from_dict(playthrough_action_dict)
-            for playthrough_action_dict in playthrough_dict[
-                'playthrough_actions']]
+        actions = [
+            LearnerAction.from_dict(action_dict)
+            for action_dict in playthrough_dict['actions']]
         return cls(
             playthrough_dict['id'],
             playthrough_dict['exp_id'],
             playthrough_dict['exp_version'],
             playthrough_dict['issue_type'],
             playthrough_dict['issue_customization_args'],
-            playthrough_actions)
+            actions)
+
+    @classmethod
+    def from_backend_dict(cls, playthrough_data):
+        """Checks whether the playthrough dict has the correct keys and then
+        returns a domain object instance.
+
+        Args:
+            playthrough_data: dict. Dict representing a playthrough.
```

Comment:
Please add expected keys with similar documentation, e.g.,

```python
playthrough_data: dict. DOCSTR.
    key1: int. DOCSTR.
    key2: str. DOCSTR.
```

Reply:
@brianrodri Seeing that we're inside the Playthrough domain class and we have access to all the fields anyway, I didn't write it. If you notice actually, its not done for any other function of the type: from_dict() either

Reply:
Resolved, will be done in separate PR.